unit-tests/src/test/scala/scala/scalanative/unsafe/CStringSuite.scala

```diff
@@ -11,29 +10,52 @@ object CStringSuite extends tests.Suite {
     fromCString(c"")
     fromCString(c"no escapes")
     fromCString(c"\'"); fromCString(c"\\'")
-    fromCString(c"\?"); fromCString(c"\\?")
+    fromCString(c"\\?")
     fromCString(c"\\")
-    fromCString(c"\a"); fromCString(c"\\a")
-    fromCString(c"\b"); fromCString(c"\\b")
-    fromCString(c"\f"); fromCString(c"\\f")
-    fromCString(c"\n"); fromCString(c"\\n")
-    fromCString(c"\r"); fromCString(c"\\r")
-    fromCString(c"\t"); fromCString(c"\\t")
-    fromCString(c"\v"); fromCString(c"\\v")
-    fromCString(c"\012\x6a")
-    fromCString(c"%s \\t %.2f %s/s\00")
+    fromCString(c"\\a")
+    fromCString(c"\\v")
+    fromCString(c"%s \\t %.2f %s/s\x00")
 
     // uncomment the following to trigger compilation errors for testing
     // fromCString(c"\")     // error at NIR
     // fromCString(c"\x2ae") // error at NIR
     // fromCString(c"\"")    // error at Scala compiler
+    // fromCString(c"\?")
+    // fromCString(c"\v")
+    // fromCString(c"\a")
+    // fromCString(c"\012")
   }
 
   test("""the value of c"..." literals""") {
+
+    assertEquals("\b", fromCString(c"\b"))
+    assertEquals("\\b", fromCString(c"\\b"))
+
+    assertEquals("\f", fromCString(c"\f"))
+    assertEquals("\\f", fromCString(c"\\f"))
+
     assertEquals("\t", fromCString(c"\t"))
     assertEquals("\\t", fromCString(c"\\t"))
-    assertEquals("\u0020\u0020\u0061\u0062", fromCString(c"\040\40\141\x62"))
-    assertEquals("\\\\v\t\\abc", fromCString(c"\\\v\t\\abc"))
+
+    assertEquals("\\n", fromCString(c"\\n"))
```

Comment:
Some people are never happy.  The added cases for \n and \r are helpful.
In super-picky mode, I did notice that the first few sets test the single backslash
case first and the double backslash case second.  Here \n & \r are done the
other way around. If this is the biggest concern I can muster, life is good,
far better than before you put this effort in.

Reply:
Looks like I introduced the \n, \\n case order inconsistency  here, or at least lead you down 
the garden path.

Mea culpa! & sorry.